packages/app-store/zapier/pages/v2/setup/index.tsx

```diff
@@ -18,7 +18,7 @@ export default function ZapierSetup(props: IZapierSetupProps) {
   const [newApiKey, setNewApiKey] = useState("");
   const { t } = useLocale();
   const utils = trpc.useContext();
-  const integrations = trpc.useQuery(["viewer.integrations", { variant: "other" }]);
+  const integrations = trpc.useQuery(["viewer.integrations", { variant: "automation" }]);
```

Comment:
There is useApp hook available now which accepts appId and tells if an app is installed or not plus gives it's data. I think that's a better way to access app data going forward.